packages/neo-one-client-core/src/user/DapiUserAccountProvider.ts

```diff
@@ -0,0 +1,427 @@
+    });
+
+    this.userAccounts$ = this.userAccountsInternal$;
+    this.currentUserAccount$ = this.currentAccountInternal$.pipe(distinctUntilChanged());
+    this.networks$ = this.networksInternal$;
+  }
+
+  public getCurrentUserAccount(): UserAccount | undefined {
+    return this.currentAccountInternal$.getValue();
+  }
+
+  public async selectUserAccount(_userAccountID?: UserAccountID): Promise<void> {
+    throw new NotImplementedError('selectUserAccount');
+  }
+
+  public async deleteUserAccount(_userAccountID?: UserAccountID): Promise<void> {
+    throw new NotImplementedError('deleteUserAccount');
+  }
+
+  public getUserAccounts(): ReadonlyArray<UserAccount> {
+    return this.userAccountsInternal$.getValue();
+  }
+
+  public getNetworks(): ReadonlyArray<NetworkType> {
+    return this.networksInternal$.getValue();
+  }
+
+  public async updateUserAccountName(_options: UpdateAccountNameOptions): Promise<void> {
+    throw new NotImplementedError('updateUserAccountName');
+  }
+
+  public async getBlockCount(network: NetworkType): Promise<number> {
+    return this.provider.getBlockCount(network);
+  }
+
+  public async getAccount(network: NetworkType, address: AddressString): Promise<Account> {
+    return this.provider.getAccount(network, address);
+  }
+
+  public iterBlocks(network: NetworkType, options?: IterOptions): AsyncIterable<Block> {
+    return this.provider.iterBlocks(network, options);
+  }
+
+  public iterActionsRaw(network: NetworkType, options?: IterOptions): AsyncIterable<RawAction> {
+    if (this.provider.iterActionsRaw === undefined) {
+      throw new NotImplementedError('iterActionsRaw');
+    }
+
+    return this.provider.iterActionsRaw(network, options);
+  }
+
+  public async transfer(
+    transfers: ReadonlyArray<Transfer>,
+    options?: TransactionOptions,
+  ): Promise<TransactionResult<TransactionReceipt, InvocationTransaction>> {
+    await this.initPromise;
+    if (transfers.length !== 1) {
+      throw new NotImplementedError('Multiple transfers per transaction');
+    }
+    const { from, networkFee, systemFee } = this.getTransactionOptions(options);
+
+    const { amount, asset, to } = transfers[0];
+
+    const { txid } = await this.dapi.send({
+      fromAddress: from.address,
+      toAddress: to,
+      asset,
+      amount: amount.toString(),
+      fee: systemFee.plus(networkFee).toString(),
+      network: from.network,
+    });
+    const transaction = (await this.provider.getTransaction(from.network, txid)) as InvocationTransaction;
+
+    return {
+      transaction,
+      confirmed: async () => this.provider.getTransactionReceipt(from.network, txid),
+    };
+  }
+
+  public async claim(_options?: TransactionOptions): Promise<TransactionResult<TransactionReceipt, ClaimTransaction>> {
+    throw new NotImplementedError('claim');
+  }
+
+  public async invokeCompleteSend(
```

Comment:
Remove this method entirely and rename `invokeCompleteSendBase` to `invokeCompleteSend`